### DISCUSSION TITLE
Is the self-hosted version behind cloud version?

### DISCUSSION THREAD

**Discussion by raabmar** [plausible/analytics#3582] [Self-Hosted Support]
I had a trial for a month and the dashboard looked differently, e.g. pin sites, show the graph of visitors in the dashboard (overview of all added sites). I struggled in the past to figure out what version my self hosted version is on (I am not planning to invest more hours just into figuring out what version I am on), but in general are self-hosted and cloud-version not the same when it comes to features?

**Comment by metmarkosaric** [comment]
Hi @raabmar! That's correct. There's a comparison with all the differences here https://plausible.io/self-hosted-web-analytics

**Comment by raabmar** [comment]
Thanks a lot - read it in the past now found it again

![2023-12-02_003802@2x](https://github.com/plausible/analytics/assets/15885290/15c3fb85-d858-464a-8659-f95d2b4b64bf)
